procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,88 @@
+# Council Synchronous Meetings
+
+This documents procedures, norms, and logistics of the Council's sync meetings.
+
+## Frequency
+
+The Council meets once a week.
+
+After the 14th of July, 2023, the Council will re-evaluate whether this frequency still makes sense or whether it should be changed.
+
+## Location
+
+The meeting happens in some video call software (e.g., Google Meet).
```

Comment:
Once this is merged, maybe we should raise an issue to get a little more concrete here. I'm not picky about the software, but it'd be good to have a default.